libs/core/src/integration/chain-event.utils.ts

```diff
@@ -399,7 +411,7 @@ const SingleContestContentUpvotedMapper: EvmMapper<
   }),
 };
 
-const EvmMappers = {
+const EvmMappers: { [key: string]: unknown } = {
```

Comment:
Should type this better. This defeats the purpose of having EVM event signatures as const. Is there a reason you don't have a mapper defined?

Reply:
migrating to evm event pattern